src/caf/toolkit/cost_utils.py

```diff
@@ -85,6 +85,46 @@ def __init__(
         self.__weighted_avg_col = weighted_avg_col
 
         self._validate_df_col_names()
+        self._validate_df_values()
+
+    def _validate_df_values(self) -> CostDistribution:
+        """sense check the values provided for the distribution"""
```

Comment:
Punctuation and additional details.
```suggestion
        """Sense check the values provided for the distribution.

        Raises
        ------
        ValueError
            If any columns contain anything other than positive numbers.

        Warns
        -----
        UserWarning
            If the bins aren't complete i.e. doesn't start at 0,
            gaps between bin edged or zero width bins.
        """
```